src/locales/fr-FR/stream.ftl

```diff
@@ -75,9 +75,9 @@ comments-rte-blockquote =
 
 comments-remainingCharacters = { $remaining } caractères restants
 
-comments-postCommentFormFake-signInAndJoin = Se connecter et rejoindre la conversation
+comments-postCommentFormFake-signInAndJoin = Se connecter et réagir
 
-comments-postCommentForm-rteLabel = Publier un commentaire
+comments-postCommentForm-rteLabel = Envoyer une réaction

```

Comment:
what's the reasoning behind changing `commentaire` to `réaction`?

Reply:
On our site a "commentaire" can be both something posted by a member to react to an article, or something posted by a journalist about a live event. So for disambiguation purpose we usually use "reaction" instead of "commentaires" when we talk about member comments. it makes sense on our site but it may not be the case everywhere

Reply:
@gfra54 I think we should leave this as "commentaire" because that's a more common usage for more newsrooms and organizations. Perhaps we could look into making this customizable later? Would you be okay with that decision?

Reply:
ok